gameobjects/cGameObject.cpp

```diff
@@ -1,17 +1,7 @@
-/*
- * cGameObject.cpp
- *
- *  Created on: 7-nov-2010
- *      Author: Stefan
- */
-
 #include "cGameObject.h"
```

Comment:
afaik we don't use `cGameObject`. it was introduced with an idea but never pursued this. We could perhaps delete this and re-introduce when really needed later?

Reply:
Sounds good. So far, there is the split between an abstract structure and abstract unit, which works fine.